modules/http1/chunk_filter.c

```diff
@@ -0,0 +1,762 @@
+/* Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Could this http1/chunk_filter.c be moved from the old one in http/ (using git mv for now and probably svn move later..)?
It'd help reviewing only the changes here, and would preserve history too.

Reply:
Giving up after several attempts. Changes to a git mv file show up as delete/new here. :(